### PR TITLE
Hotfix/fix malformed input exception

### PR DESCRIPTION
**Pull request by gaabs**
This aims to solve  #71

### CODE REVIEW

File: src/main/java/br/ufpe/cin/files/FilesManager.java [hunk 1]
```diff
@@ -313,9 +314,7 @@ public static boolean writeContent(String filePath, String content){
 					file.getParentFile().mkdirs();
 					file.createNewFile();
 				}
-				BufferedWriter writer = Files.newBufferedWriter(Paths.get(filePath));
-				writer.write(content);
-				writer.flush();	writer.close();
+				FileUtils.write(file, content);
 			} catch(NullPointerException ne){
```
Comment (pauloborba) [inline src/main/java/br/ufpe/cin/files/FilesManager.java]: @jvcoutinho hasn't these kinds of issues been resolved before?

Comment (jvcoutinho) [reply src/main/java/br/ufpe/cin/files/FilesManager.java]: Well, I haven't detected this specific one, but I don't know if my solutions cover this, maybe. Anyway, they haven't been integrated yet and FileUtils seems better to use.
It also looks like an encoding problem.

Comment (pauloborba) [reply src/main/java/br/ufpe/cin/files/FilesManager.java]: @jvcoutinho could you then please fix that?

**Comment by guilhermejccavalcanti** [comment]
@gaabs  e @jvcoutinho conflito nos binários

**Pull request merged**